docs/FAQ.md

```diff
@@ -0,0 +1,51 @@
+# SDG FAQs
+
+## Where can I find more details about the synthetic data generation algorithm?
+
+The [Large-Scale Alignment for ChatBots](https://arxiv.org/pdf/2403.01081) research paper does an excellent job of explaining the SDG process in section 3.2.
+
+## How do I estimate the amount of samples a given leaf node will produce in the SDG process for training?
+
+For each question and answer pair in the taxonomy leaf node in a skill file: an estimated 30 synthetic samples will be produced in the training dataset.  
+
+For each knowledge leaf node: the formula to estimate the number of produced synthetic samples in the training dataset is:
+
+```text
+(total cumulative size of knowledge documents / max document chunk size) * number of qna pairs in the knowledge file leaf node * 30 synthetic samples per qna pair
+```
+
+For example: let’s say the total size of the knowledge markdown files in the knowledge directory are 1 MB in size and there are 15 question and answer pairs in the knowledge leaf node file. You can estimate the total number of knowledge synthetic samples for that leaf node to be:
+
+```text
+1MB * 1048576 bytes / 4 bytes per token / 1.3 tokens per word / 1000 word document chunks ~= 202 chunks
+
+202 chunks * 15 qna pairs * 30 samples per pair = 90900 samples
+```
+
+Note that this is just an estimation technique: the exact number is non-deterministic since the results are ultimately evaluated against a judge model and results that do not pass a score metric are dropped from the overall set.
+
+## How many seed_examples can Instructlab process in a knowledge leaf node
+
+There is no known limit to the number of seed example entries for a knowledge leaf node. There must be a minimum of 5 seed examples. These parameters can be seen in the taxonomy schema repo:  <https://github.com/instructlab/schema/blob/main/src/instructlab/schema/v3/knowledge.json>
+
+## How many qna pairs can be listed for a given seed_example in a knowledge leaf node
+
+**Exactly** 3 QNA pairs must be listed for a given seed_example. If more is specified they will be ignored and not processed by the appropriate prompt. This can be seen by looking at the prompt files in SDG: <https://github.com/instructlab/sdg/blob/v0.6.2/src/instructlab/sdg/configs/knowledge/simple_generate_qa.yaml#L21-L28>
```

Comment:
I'm a bit uncomfortable linking to v0.6.2 here directly, as this doc lives in the main branch. It feels like we do a relative link (can we do that in github markdown?) to the file in the same branch as this doc.